packages/ens-controller/package.json

```diff
@@ -30,8 +30,8 @@
   "dependencies": {
     "@ethersproject/providers": "^5.7.0",
     "@metamask/base-controller": "^3.2.1",
-    "@metamask/controller-utils": "^4.3.2",
-    "@metamask/network-controller": "^12.2.0",
+    "@metamask/controller-utils": "^5.0.0",
```

Comment:
ENSController consumes controller-util 5.0.0 changes. Does this need a changelog entry?